Test/Unit/Model/Config/GridXmlToArrayConverterTest.php

```diff
@@ -420,13 +424,37 @@ private function getMassActionXml(): string
     private function getMassActionExpected(): array
     {
         $actions = [
-            ['label' => 'Update', 'url' => '*/massActions/update'],
-            ['label' => 'Delete All', 'url' => '*/massActions/delete', 'requireConfirmation' => true],
-            ['label' => 'Reindex', 'url' => '*/massActions/reindex'],
+            ['id' => 'update', 'label' => 'Update', 'url' => '*/massActions/update'],
+            ['id' => 'delete','label' => 'Delete All', 'url' => '*/massActions/delete', 'requireConfirmation' => true],
+            ['id' => 'reindex','label' => 'Reindex', 'url' => '*/massActions/reindex'],
```

Comment:
Why add id's to mass actions in the scope of this PR?

Reply:
it is acutally not in the scope of this pr and sorry that i mixed it up. I started working with the module and then figured out that i would like to do the PR. Hope that works for you though.